src/Symfony/Bundle/SecurityBundle/Command/UserPasswordEncoderCommand.php

```diff
@@ -75,29 +82,38 @@ protected function execute(InputInterface $input, OutputInterface $output)
         $this->writeIntroduction($output);
 
         $password = $input->getArgument('password');
-        $salt = $input->getArgument('salt');
         $userClass = $input->getArgument('user-class');
+        $emptySalt = $input->getOption('empty-salt');
 
         $helper = $this->getHelper('question');
 
         if (!$password) {
+            if (!$input->isInteractive()) {
+                throw new \Exception('The password must not be empty.');
```

Comment:
Why not? I know that it's an edge case, but someone would like to encode a blank password (using some random salt).